cmake/app.cmake

```diff
@@ -23,7 +27,6 @@ endif()
 
 # Include Mbed OS main cmake
 add_subdirectory(mbed-os)
-
```

Comment:
Why remove this newline in commit "Add target definitions and properties from the tools"? I think we should keep the newline.

Reply:
Newline restored.